src/sessions/video-session-handler.ts

```diff
@@ -255,6 +282,13 @@ export class VideoSessionHandler extends BaseSessionHandler {
       return;
     }
 
+    if (this.requestedMeetingSessions[pendingSession.meetingId]) {
+      logPendingSession(this.sdk.logger, 'Propose received for requested video session, accepting automatically', pendingSession, 'debug');
```

Comment:
I'd like to also unique-ify this log message.

Reply:
Do you think simply adding conference/meeting would be sufficient? Something like `Propose received for requested conference/meeting video session, accepting automatically`?

Whatever we go with, I'll try to have some consistency between above and here.